triage.yaml

```diff
@@ -0,0 +1,17 @@
+name: Check Triage Status of Issue
+on: 
+  issues:
+    types: [opened, closed, reopened, transferred, labeled, unlabeled]
+    # Issue is created, Issue is closed, Issue added or removed from projects, Labels added/removed
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Kanban
+        uses: machine-learning-apps/kubeflow-triage@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It looks like you are setting the environment variable GITHUB_TOKEN here. So can't we just set it to the secret containing the GITHUB_TOKEN to modify the kanban board? I.e why do we need to change the code to use a different environment variable then the name GITHUB_TOKEN?

Reply:
No there is another variable called GITHUB_TOKEN that you cannot override in the Actions environment, this is why I had to change the name of the environment variable.  This diff is a bit old I updated the interface PTAL